docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -12,6 +12,19 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/react-sdk                   | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2023-03-14
+
+Released: `@100mslive/hms-video-store@0.9.0`, `@100mslive/react-sdk@0.7.0`
+
+#### Added:
+- `getAuthTokenByRoomCode` in hmsActions to get token for room short code.
+
+#### Fixed:
+- `Uncaught exceptions` are handled properly for most cases.
+
+#### Breaking:
+- Video attach/detach and auto layer selection is moved to core sdk now. Earlier this was only available in `useVideo` hook of `react-sdk`.
```

Comment:
this should be in Added section.

- The SDK now handles auto subscribing/unsubscribing to video [based on the element's visibility](https://www.100ms.live/docs/javascript/v2/how--to-guides/set-up-video-conferencing/render-video/overview#detach-video-to-conserve-bandwidth-and-cleanup-elements). If Simulcase is enabled, the appropriate resolution based on the video element's dimensions will also be auto-selected. Note that this was already available for react-sdk, with this release, we've also added it for the core SDK.